### DISCUSSION TITLE
Lit implementation on Payload

### DISCUSSION THREAD

**Discussion by rrums123** [payloadcms/payload#2109] [Q&A]
Hey there, I want to use Lit on React for `ui` field on payload. I use [`@lit-labs/react`](https://www.npmjs.com/package/@lit-labs/react) to do it.

I got error like this
```
Error [ERR_REQUIRE_ESM]: require() of ES Module /path/payload/node_modules/lit-element/index.js from /path/payload/src/lit-components/test/my-element.js not supported.

Instead change the require of index.js in /path/payload/src/lit-components/test/my-element.js to a dynamic import() which is available in all CommonJS modules.
```

Is there any way to implement Lit in Payload?

**Comment by jmas** [comment]
Hello, I guess here you can find more information about the similar issue:
* https://stackoverflow.com/questions/69081410/error-err-require-esm-require-of-es-module-not-supported
* https://stackoverflow.com/questions/69041454/error-require-of-es-modules-is-not-supported-when-importing-node-fetch

**Reply by rrums123** [reply]
Yeah, but different case, because node-fetch v2 use cjs, but (I think) Lit do not have any version that use cjs. So maybe use bundler will be an answer. I'm open this discussion maybe if someone has an attempt to implement Lit and share their experience, that will be useful.
